src/Germany/Mews.Fiscalizations.Germany.Tests/TestFixture.cs

```diff
@@ -1,4 +1,5 @@
-﻿using Mews.Fiscalizations.Germany.Model;
+﻿using Mews.Fiscalizations.Germany.V1;
+using Mews.Fiscalizations.Germany.V1.Model;
```

Comment:
This is weird. Why do we now need more usings? :D

Reply:
`FiskalyClient` is no longer in Mews.Fiscalizations.Germany
it is now Mews.Fiscalizations.Germany.V1 so we must add the using.

the other using was there before.